packer.mk

```diff
@@ -1,6 +1,7 @@
 SHELL=/bin/bash -o pipefail
 
-build-packer:
+build-packer: import-image
+ifeq (${OS_DISTRO_FAMILY},Ubuntu)
```

Comment:
Ubuntu doesn't have to import image(and it shouldn't )

Reply:
Removed import-image as a makefile target and just put it into the Mariner condition of build-packer.